cluster/addons/cluster-monitoring/heapster-rbac.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: system:heapster-cluster-role
```

Comment:
> @deads2k thoughts on system: prefixes for roles defined in addons?

I'm ok with doing it for addons which live in the kubernetes org (not kubernetes-incubator).